t/t1002-mf-priority-small-no-tie.t

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+test_description='Test multi-factor priority plugin order with no ties'
+
+. `dirname $0`/sharness.sh
+MULTI_FACTOR_PRIORITY=${FLUX_BUILD_DIR}/src/plugins/.libs/mf_priority.so
+SUBMIT_AS=${FLUX_BUILD_DIR}/t/scripts/submit_as.py
+
+export TEST_UNDER_FLUX_NO_JOB_EXEC=y
+export TEST_UNDER_FLUX_SCHED_SIMPLE_MODE="limited=1"
```

Comment:
I'll have to look into it further to be sure, but I don't think you need to set these since you are "stopping" the queue for the duration of the tests.